app/bot/config.yml

```diff
@@ -19,43 +19,47 @@ commands:
     content: |
       **Looking for information on Migration to v1?**
 
-      [Beta Release Discussion](https://github.com/hay-kot/mealie/discussions/1073)
+      [Beta Release Discussion](https://github.com/mealie-recipes/mealie/discussions/1073)
       [v0.5.x to v1.0.0 Migration Guide](https://nightly.mealie.io/documentation/getting-started/migrating-to-mealie-v1/)
+      [Migrating from other V1 versions](https://nightly.mealie.io/documentation/getting-started/installation/installation-checklist/#migrating-from-other-v1-versions)
 
   - command: "mealie-docker-faq"
     description: "Show helpful Docker Problems and Solutions"
     content: |
       **Having trouble with docker?**
 
-      1. Is your API_URL set correctly on the Frontend container?
-      2. Have you verified your volumes are configured correctly?
-      3. Have you reviewed the installation checklist?
+      1. Have you verified your volumes are configured correctly?
```

Comment:
You can remove this, it isn't really relevant anymore now that we don't have the shared volume thing.

Reply:
Right, I missed that!